demisto_sdk/commands/validate/tests/PB_validators_test.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from demisto_sdk.commands.validate.validators.PB_validators.PB118_is_input_key_not_in_tasks import (
+    IsInputKeyNotInTasksValidator,
+)
+from demisto_sdk.commands.validate.tests.test_tools import create_playbook_object
+
+
+playbook = create_playbook_object()
+
+
+@pytest.mark.parametrize(
+    "input_in_task,input_in_section, expected_result",
+    [
+        ({"input1", "inputs3"}, {"input1"}, []),
+        (
+            {"input1", "inputs3"},
+            {"input1", "input2"},
+            "The playbook 'Phishing Investigation - Generic' contains the following inputs that are not used in any of its tasks: input2",
+        ),
+    ],
```

Comment:
It would be better to use the **create_playbook_object** here to create multiple playbook items and test them.
Let's make sure we cover all cases here.

Reply:
Done